web/static/win-stats.mjs

```diff
@@ -14,6 +14,7 @@ class WinStats extends LitElement {
 
     async loadData() {
         this.data = await RPCCall('WinStats');
+        setTimeout(() => this.loadData(), 10 * 60 * 1000); // 10 minutes
```

Comment:
10 min in a little high, you don't win that often, but I'd still lower this to maybe 2 minutes